_caster.py

```diff
@@ -41,3 +70,14 @@
     sikuli_controller.get_instance().bootstrap_start_server_proxy()
 
 print("\n*- Starting " + settings.SOFTWARE_NAME + " -*")
+
+try:
+    imp.find_module('PySide2')
```

Comment:
looks like `imp` has deprecated in Py v3.4. `importlib.import_module` looks like it's an alternative.

`importlib.import_module('PySide2')`

Reply:
However to maintain Python to compatibility `imp` should be used.